core/domain/exp_jobs_one_off.py

```diff
@@ -57,6 +60,32 @@ def map(exploration_model):
     def reduce(exp_id, list_of_exps):
         pass
 
+class ExpSummariesContributorsOneOffJob(jobs.BaseMapReduceJobManager):
+    """ One-off job that finds the user id's of the contributors
+    (defined as any human who has made a positive (i.e. not just
+    a reversion) for each exploration.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationSnapshotMetadataModel]
+
+    @staticmethod
+    def map(item):
+        _VERSION_DELIMITER = '-'
+        _COMMIT_TYPE_REVERT = 'revert'
+        if item.commit_type != _COMMIT_TYPE_REVERT:
+            if item.committer_id not in feconf.SYSTEM_USER_IDS:
+                snapshot_id = item.id
+                exp_id = snapshot_id[:snapshot_id.rfind(_VERSION_DELIMITER)]
+                yield (exp_id, item.committer_id)
+
+    @staticmethod
+    def reduce(exp_id, committer_id_list):
+        committer_ids = set(committer_id_list)
```

Comment:
So ast.literal_eval interprets whatever is passed into it as a python value-- what I'm getting here is already a perfectly valid list of strings, so I'm just using that instead of calling literal_eval. Interestingly, literal_eval was interpreting the numeric strings as longs-- the id's returned by "get_user_id_from_email" are numeric.